spec/formatting.md

```diff
@@ -237,6 +237,21 @@ _Pattern selection_ is not supported for _fallback values_.
 
 ## Pattern Selection
 
+### Messages without Selectors
+
+When a _message_ does not contain a _match_ construct,
+and does not begin with a U+007B LEFT CURLY BRACKET `{`,
+the result of pattern selection MUST be the concatenation of
+a U+007B LEFT CURLY BRACKET `{`,
+the contents of the message as a string,
+and a U+007D RIGHT CURLY BRACKET `}`.
+
+> For example, the following message:
+> `no braces`
+> would format to a string as `{no braces}`.
```

Comment:
I'm not sure this is "pattern selection". This is a parse error, since the message is not, well, a message. What's more, the message "no braces {some braces}" is not a message.

I think the thing that is missing in `formatting.md` is the first step, which is: parsing. This should result in an syntax error. We could then go on to define what happens to the string that wants to be a message. 

To be honest, I would not add the curly brackets. If you give me a string and it produces a syntax error, I should probably give you back the string as you gave it to me. If you thought it should be a valid message, you can debug it. If you are a runtime process, you can just display it (as we didn't damage it).

---

Note well: I argued long and hard that a plain old string with no placeholders should be a valid message. This is the old argument code-mode vs. string-mode. I am not reopening this, but we're going to tangle with the fact that you have to grope strings before feeding them to MF2 as a result.